src/components/app-switcher/app-switcher.scss

```diff
@@ -0,0 +1,98 @@
+@import "./../../style-config/colors";
+.carbon-icon__svg--grid {
```

Comment:
If some of this CSS relates to how the icon is rendered with the context of the app switcher, then it should be nested with the appropriate class, otherwise it will act as a global style.